PAM_app/app/src/main/java/com/example/pam_app/foodAndDrinks/FoodModel.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.pam_app.foodAndDrinks
+
+class FoodModel(image: Int?, name: String?, price: String?) {
+
+     var setGetImage: Int? = image
+        get() = field
+        set(value) {
+            field = value
+        }
+    var setGetName: String? = name
+        get() = field.toString()
+        set(value) {
+            field = value.toString()
+        }
+    var setGetPrice: String? = price
+        get() = field.toString()
+        set(value) {
+            field = value.toString()
+        }
```

Comment:
Change name of variable to image, name and price. There is no point to use such names as `setGet`